packages/evm-contracts/contracts/IValist.sol

```diff
@@ -6,49 +6,60 @@ interface IValist {
   /// @dev emitted when a new team is created
   event TeamCreated(string _teamName, string _metaURI, address _sender);
   /// @dev emitted when an exsting team is updated
-  event TeamUpdated(string _teamName, string _metaURI, address _member);
+  event TeamUpdated(string _teamName, string _metaURI, address _sender);
   /// @dev emitted when a new team member is added
-  event TeamMemberAdded(string _teamName, address _member);
+  event TeamMemberAdded(string _teamName, address _member, address _sender);
   /// @dev emitted when an existing team member is removed
-  event TeamMemberRemoved(string _teamName, address _member);
+  event TeamMemberRemoved(string _teamName, address _member, address _sender);
+
+  /// @dev emitted when a beneficiary is updated on a license.
+
+  event BeneficiaryUpdated(
```

Comment:
`TeamBeneficiaryUpdated` would be more consistent

Reply:
updated!